Apps/Contoso.WPF.Demo/App.xaml.cs

```diff
@@ -13,10 +14,84 @@ namespace Contoso.WPF.Demo
     /// </summary>
     public partial class App
     {
+        private const string LogTag = "AppCenterDemo";
+
         protected override void OnStartup(StartupEventArgs e)
         {
             AppCenter.LogLevel = LogLevel.Verbose;
+            Crashes.ShouldAwaitUserConfirmation = ConfirmationHandler;
+            Crashes.ShouldProcessErrorReport = ShouldProcess;
+
+            // Event handlers.
+            Crashes.SendingErrorReport += SendingErrorReportHandler;
+            Crashes.SentErrorReport += SentErrorReportHandler;
+            Crashes.FailedToSendErrorReport += FailedToSendErrorReportHandler;
             AppCenter.Start("f4e2a83d-3052-4884-8176-8b2c50277d16", typeof(Analytics), typeof(Crashes));
+            Crashes.HasCrashedInLastSessionAsync().ContinueWith(hasCrashed =>
+            {
+                AppCenterLog.Info(LogTag, "Crashes.HasCrashedInLastSession=" + hasCrashed.Result);
```

Comment:
Please do not use `AppCenterLog` in Demo and Test app. This should use its own logging framework.